.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules/
 data
 data_old
 package-lock.json
+launch_demo.sh
```

Comment:
Ideally you shouldn't add things that don't concern other developers to gitignore, you could either move the file somewhere else or ignore it locally following https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally